detailedSearch.php

```diff
@@ -685,13 +685,23 @@ class GameResultData
 		if ($seeBronze=='checked') { print '<th class= "advancedSearch">IsBronze</th>'; }
 
 		print "</tr>";
-
+		global $User;
```

Comment:
Can you move this to the top of the function just to match how it's used everywhere else?